docker-compose.yml

```diff
@@ -6,12 +6,12 @@ services:
       context: .
     network_mode: host
     ports:
-      - '8000:8000'
+      - '8000:7766'
```

Comment:
there's a conflict if it's run on port 8000 in the container, and it's not clear why that happens. Normally this isn't an issue.